src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/ComboBoxSamplePage.xaml

```diff
@@ -63,6 +63,40 @@
 			<sample:SamplePageLayout.CupertinoTemplate>
 				<DataTemplate>
 					<StackPanel>
+						<StackPanel BorderBrush="{StaticResource CupertinoOpaqueSeparatorBrush}"
+									BorderThickness="1"
+									android:Margin="12"
+									ios:Margin="12"
+									Padding="12">
+
+							<!-- Label -->
+							<TextBlock Text="NORMAL"
+									   Style="{StaticResource TertiaryTitleTextBlockStyle}" />
+
+							<!-- ComboBox -->
+							<smtx:XamlDisplay UniqueKey="Cupertino_ComboBoxSamplePage_Normal"
+											  Style="{StaticResource XamlDisplayBelowStyle}">
+								<ComboBox Style="{StaticResource CupertinoComboBoxStyle}"
+										  Text="Title"
+										  PlaceholderText="Placeholder"
+										  ItemsSource="abcdefghijklmn" />
+							</smtx:XamlDisplay>
+
+							<!-- Label -->
+							<TextBlock Text="DISABLED"
+									   Margin="0,10,0,0"
+									   Style="{StaticResource TertiaryTitleTextBlockStyle}" />
```

Comment:
@kazo0 Given that material textblock styles are `Material_` prefixed, shouldn't the cuppertino ones receive the same treatment as well?

Reply:
Yeah I have a branch that I'm waiting to push that renames the textblock style names, waiting for this to merge first